### PR TITLE
Store generated object field value to generate depended fields - example for Personality.age

### PR DESCRIPTION
**Pull request by yn-coder**
Store generated object field value to generate depended fields - example for Personality.age

https://github.com/lk-geimfari/elizabeth/issues/107

With test coverage

### CODE REVIEW

File: elizabeth/core/elizabeth.py [hunk 1]
```diff
@@ -815,8 +816,42 @@ def age(minimum=16, maximum=66):
         :Example:
             23.
         """
-        return randint(minimum, maximum)
+        a = randint(minimum, maximum)
+        self._store['age'] = a
+        return a
 
+    def child_count( self, max_childs = 5 ):
```
Comment (lk-geimfari) [inline elizabeth/core/elizabeth.py]: Should be: `def child_count(self, max_childs=5):`.

File: elizabeth/core/elizabeth.py [hunk 2]
```diff
@@ -815,8 +816,42 @@ def age(minimum=16, maximum=66):
...
+        if a == 0:
+            a = self.age()
+        
+        if a < 18:
+            cc = 0 # underage
+        else:
+            cc = randint( 0, max_childs )
+        return cc
+    
+    def work_experience( self, working_start_age = 22 ):
```
Comment (lk-geimfari) [inline elizabeth/core/elizabeth.py]: Here too.

File: elizabeth/core/elizabeth.py (hunk 1 above)
Comment (lk-geimfari) [inline elizabeth/core/elizabeth.py]: and here

**Comment by lk-geimfari** [comment]
Read about PEP8, please. Merged, but in future you should know about [PEP8](https://www.python.org/dev/peps/pep-0008/).

**Comment by yn-coder** [comment]
You mean `too many blancks` ?

**Comment by lk-geimfari** [comment]
@yn-coder Yes. I recommend you really good [guide](http://docs.python-guide.org/en/latest/).

**Pull request merged**